src/NzbDrone.Core/Notifications/Telegram/Telegram.cs

```diff
@@ -33,12 +34,16 @@ public override void OnEpisodeFileDelete(EpisodeDeleteMessage deleteMessage)
 
         public override void OnSeriesAdd(SeriesAddMessage message)
         {
-            _proxy.SendNotification(SERIES_ADDED_TITLE, message.Message, Settings);
+            var text = FormatMessageWithLink(message.Message, message.Series);
```

Comment:
These links should also be included for `OnGrab`, `OnDownload` and `OnEpisodeFileDelete`